promql/functions.go

```diff
@@ -1385,22 +1386,40 @@ func init() {
 		delete(parser.Functions, "xincrease")
 		delete(parser.Functions, "xrate")
 		fmt.Println("Successfully replaced rate & friends with xrate & friends (and removed xrate & friends function keys).")
-	case "2":
-		FunctionCalls["delta"] = FunctionCalls["ydelta"]
-		parser.Functions["delta"] = parser.Functions["ydelta"]
-		parser.Functions["delta"].Name = "delta"
 
-		FunctionCalls["increase"] = FunctionCalls["yincrease"]
-		parser.Functions["increase"] = parser.Functions["yincrease"]
-		parser.Functions["increase"].Name = "increase"
-
-		FunctionCalls["rate"] = FunctionCalls["yrate"]
-		parser.Functions["rate"] = parser.Functions["yrate"]
-		parser.Functions["rate"].Name = "rate"
-		fmt.Println("Successfully replaced rate/increase/delta with yrate/yincrease/ydelta (and left the latter names available as well).")
+	case "x", "X":
+		copy_parser_function("delta", "orig_delta")
+		copy_parser_function("increase", "orig_increase")
+		copy_parser_function("rate", "orig_rate")
+		repoint_function("delta", "xdelta", "orig_delta")
+		repoint_function("increase", "xincrease", "orig_increase")
+		repoint_function("rate", "xrate", "orig_rate")
+
+		fmt.Println("Successfully replaced rate/increase/delta with xrate/xincrease/xdelta (and left the latter names available as well, and original functions callable as orig_rate/orig_increase/orig_delta).")
+
+	case "2", "y", "Y":
+		copy_parser_function("delta", "orig_delta")
+		copy_parser_function("increase", "orig_increase")
+		copy_parser_function("rate", "orig_rate")
+		repoint_function("delta", "ydelta", "orig_delta")
+		repoint_function("increase", "yincrease", "orig_increase")
+		repoint_function("rate", "yrate", "orig_rate")
+
+		fmt.Println("Successfully replaced rate/increase/delta with yrate/yincrease/ydelta (and left the latter names available as well, and original functions callable as orig_rate/orig_increase/orig_delta).")
 	}
 }
 
+func copy_parser_function(from_name string, to_name string) {
```

Comment:
Style nit: the function and parameter names should be in snakeCase.

Reply:
Fixed: https://github.com/Invoca/prometheus/pull/5/commits/d691b25f92d0b24ea91e325d72ef6db1109bdf65